lib/utils.js

```diff
@@ -134,7 +134,7 @@ function parseStack(err, cb) {
     var frame = {
         filename: line.getFileName() || '',
         lineno: line.getLineNumber(),
-        'function': getFunction(line),
+        function: getFunction(line),
```

Comment:
I think it's safer to leave this (notice it's highlighted as a keyword here). Revert this one line and I'll merge.

Thanks!